src/main/java/seedu/address/logic/commands/AddAssignmentCommand.java

```diff
@@ -0,0 +1,91 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_ASSIGNMENT;
+import static seedu.address.model.Model.PREDICATE_SHOW_ALL_PERSONS;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.person.Assignment;
+import seedu.address.model.person.Person;
+
+/**
+ * Adds an assignment to all students in the address book.
+ */
+public class AddAssignmentCommand extends Command {
+
+    public static final String COMMAND_WORD = "addAssignment";
```

Comment:
would `adda` be better? since it'll be easier and faster for users to type out also